src/request-factory.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+var ResultList = require('./result-list');
+
+function RequestFactory(retriever) {
+  this._retriever = retriever;
+}
+
+RequestFactory.prototype.get = function(options) {
+  return this.resultList(options).next();
```

Comment:
I take that back, it will be defined. It is still a little weird, but this will work.